File: examples/php/constructs/type_annotations.php
<?php
// Ported from examples/lumen/constructs/type_annotations.lm by scripts/port_examples.py; edit the Lumen original, not this file.
print("Test: Type Annotations on Variables\n");
$x = 42;
print("let x: number = 42\n");
print($x . "\n");
$message = "Hello, World";
print("let message: string = \n");
print($message . "\n");
$flag = true;
print("let flag: boolean = true\n");
print($flag . "\n");
$empty = null;
print("let empty: null = null\n");
print($empty . "\n");
function add($a, $b) {
    return $a + $b;
}

print("add(5, 3):\n");
print(add(5, 3) . "\n");
function greet($name) {
    return "Hello, " + $name;
}

print("greet(\"Alice\"):\n");
print(greet("Alice") . "\n");
function process($x, $y) {
    $result = $x * 2 + $y;
    return $result;
}

print("process(10, 5):\n");
print(process(10, 5) . "\n");

File: examples/python/constructs/type_annotations.py
# Ported from examples/lumen/constructs/type_annotations.lm by scripts/port_examples.py; edit the Lumen original, not this file.
print("Test: Type Annotations on Variables")
x = 42
print("let x: number = 42")
print(x)
message = "Hello, World"
print("let message: string = ")
print(message)
flag = True
print("let flag: boolean = true")
print(flag)
empty = None
print("let empty: null = null")
print(empty)
def add(a, b):
    return a + b

print("add(5, 3):")
print(add(5, 3))
def greet(name):
    return "Hello, " + name

print("greet(\"Alice\"):")
print(greet("Alice"))
def process(x, y):
    result = x * 2 + y
    return result

print("process(10, 5):")
print(process(10, 5))
